archetypes/package-release.md

```diff
@@ -0,0 +1,39 @@
+---
+title: ''
+description: >
+    A 2-3 sentence description of the post that appears on the articles page.
+date: {{ .Date }}
+author: Hadley Wickham
+# Featured photo
+photo:
+  url: https://unsplash.com/photos/n6vS3xlnsCc
+  author: Kelley Bozarth
+# one of: "case studies", "learn", "package", "programming", or "other"
+categories: [package] 
+---
+
+
+```{r setup, include = FALSE}
+library(testthat) # add / replace as needed
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
```

Comment:
@jennybc, do you think we should add a placeholder here for "You can install {insert package here} from CRAN…" plus installation code chunk (with `eval = FALSE`) here?

Reply:
Yeah. There are several recent-ish posts where a template could be found or at least derived. Note that many other posts *don't* do this 😕

I think every package release post should include a short standard description of what the package does, how to install it from CRAN, and how to attach via `library()`.

The principle is there shouldn't be some mission critical single line of code that we systematically hide and without this implicit knowledge, nothing in the blog post would actually run for a reader.